### PR TITLE
TASK-1100

### PR DESCRIPTION
**Pull request by pfurio**
TASK-1100

### CODE REVIEW

**Comment by pfurio** [comment]
Task linked: [TASK-1100 Project configured cellbase being ignored in VariantStorageManager](https://app.clickup.com/t/36631768/TASK-1100)

**Review by j-coll — CHANGES_REQUESTED** [review]
Take a look to this code:
[VariantStorageManager:989-995](https://github.com/opencb/opencga/blob/v2.3.0/opencga-analysis/src/main/java/org/opencb/opencga/analysis/variant/manager/VariantStorageManager.java#L989-L995)

 The "include" will not include the cellbase object.

**Review by imedina — CHANGES_REQUESTED** [review]
@pfurio CellBase in Azure is v5 not v4

**Comment by pfurio** [comment]
Changes made. Please review again when you can @imedina @j-coll

File: opencga-core/src/main/java/org/opencb/opencga/core/models/project/ProjectUpdateParams.java [hunk 1]
```diff
@@ -112,12 +107,4 @@ public ProjectUpdateParams setAttributes(Map<String, Object> attributes) {
         return this;
     }
 
-    public CellBaseConfiguration getCellbase() {
```
Comment (j-coll) [inline opencga-core/src/main/java/org/opencb/opencga/core/models/project/ProjectUpdateParams.java:115]: You need to regenerate the command line. It doesn't compile now

**Review by j-coll — CHANGES_REQUESTED** [review]

**Review by j-coll — APPROVED** [review]

**Pull request merged**